lib/compat/wordpress-6.8/rest-api.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * PHP and WordPress configuration compatibility functions for the Gutenberg
+ * editor plugin changes related to REST API.
+ *
+ * @package gutenberg
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'Silence is golden.' );
+}
```

Comment:
Why is this here? It doesn't seem like something we do in this plugin.

Reply:
We have referenced the rest-api.php file from WordPress 6.6.